src/app/components/home/TransactionPanel.tsx

```diff
@@ -348,30 +307,11 @@ const getAmountWithSign = (tx: Transaction) => {
     }
 
     case 'Claimed': {
-      const {
-        amount1,
-        amount2,
-        amount3,
-        amount4,
-        amount5,
-        amount6,
-        symbol1,
-        symbol2,
-        symbol3,
-        symbol4,
-        symbol5,
-        symbol6,
-      } = getValuesAndSymbols(tx);
-      return (
-        <>
-          <AmountItem value={amount1} symbol={symbol1} positive={true} />
-          <AmountItem value={amount2} symbol={symbol2} positive={true} />
-          <AmountItem value={amount3} symbol={symbol3} positive={true} />
-          <AmountItem value={amount4} symbol={symbol4} positive={true} />
-          <AmountItem value={amount5} symbol={symbol5} positive={true} />
-          <AmountItem value={amount6} symbol={symbol6} positive={true} />
-        </>
-      );
+      const amountItemList = getValuesAndSymbols(tx);
```

Comment:
There are two `Claimed` cases. Please merge the logic in one case.

Reply:
@octocat512 one for getContent() one for getValuesAndSymbols(). Please check again. Thank you!

Reply:
haha. my bad. @Chunn